### PR TITLE
UHM-6643: Lab OWA Assessments and Tweaks

### PR DESCRIPTION
**Pull request by mogoodrich**
Add support for filtering by tests nested within panels

### CODE REVIEW

File: app/js/components/LabResultsList/LabResultsTable.jsx [hunk 1]
```diff
@@ -33,13 +33,19 @@ function filterData(filters, data) {
...
+      categories.forEach((category) => {
+        if (category.setMembers && category.setMembers.length > 0) {
+          category.setMembers.forEach((testOrPanel) => {
+            testFilter.push(testOrPanel.uuid);
+            // handle panels, which will have nested tests
+            if (testOrPanel.setMembers && testOrPanel.setMembers.length > 0) {
+              testOrPanel.setMembers.forEach((test) => {
+                testFilter.push(test.uuid)
+              })
+            }
```
Comment (mogoodrich) [inline app/js/components/LabResultsList/LabResultsTable.jsx:48]: Just added support for going a level deeper, so we can filter by tests within panels.  Complicates things, but I tweaked the variable names to hopefully keep it easy-ish to understand.

**Review by cioan — APPROVED** [review]
Looks good, @mogoodrich ! Thanks!

**Comment by mogoodrich** [comment]
Thanks @cioan ... @mseaton  I'm merging this in so I can start working on the next part of this ticket and base it off an up-to-date master, but post-commit reviews welcome.

**Review by mseaton — COMMENTED** [review]
LGTM - naming much clearer, thanks.

**Pull request merged**